packages/roc-package-web-app-react/package.json

```diff
@@ -31,27 +31,22 @@
   "dependencies": {
     "debug": "~2.2.0",
     "error": "~7.0.2",
-    "history": "~1.17.0",
-    "nunjucks": "~2.1.0",
-    "pretty-error": "~1.2.0",
-    "react-a11y": "~0.2.6",
-    "react-fetcher": "~0.2.0",
+    "history": "^2.1.0",
```

Comment:
Yes, we need it because we are using a custom `history`. 

https://github.com/rocjs/roc-package-web-app-react/blob/feature/react-15/packages/roc-package-web-app-react/app/client/create-client.js#L8